clients/rust/client/src/lib.rs

```diff
@@ -0,0 +1,17 @@
+#![forbid(unsafe_code)]
+#![deny(missing_docs)]
+
+//! # Gamma Rust client library
```

Comment:
A small example how of to use the library would be very nice here

Reply:
That is true! In fact, I think multiple examples could be useful as it works differently for the API part and Oauth2 part.